devenv.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs, lib, config, inputs, ... }:
+
+{
+  packages = [ pkgs.git ];
+
+  languages.javascript = {
+    enable = true;
+    yarn.enable = true;
+  };
+
+  enterShell = ''
+    git --version
+    node --version
+    yarn install
```

Comment:
@sadjow I believe there is an option for this:

https://devenv.sh/reference/options/#languagesjavascriptyarninstallenable